File: Menú.py
print ("¡Bienvenido!")
print ("Ingrese el numero de la izquierda acorde a la opción que desea")
print ("1. Desayuno", " ")
print ("2. Almerzo", " ")
print ("3. Cena", " ")
print ("4. Postres", " ")
print ("5. Antojos", " ")
print ("6. Salir", " ")
print (" ")
eleccion = int(input("Respuesta: "))
print (" ")

if eleccion == 1:
    print ("Las opciones de desayuno son las siguientes: ", " ")
    print ("1. Pupusas", " ")
    print ("2. Huevos con frijoles", " ") 
    print ("3. Hot Cakes", " ")
    print (" ")
    comida = int(input("Respuesta: "))
    print (" ")
    if comida == 1:
        print ("Su orden es: Pupusas")
    elif comida ==2:
        print ("Su orden es: Huevos con frijoles")
    elif comida ==3:
        print ("Su orden es: Hot Cakes")
    else: print ("Valor ingresado incorrecto")

elif eleccion == 2:
    print ("Las opciones de almuerzo son las siguientes: ", " ")
    print ("1. Carne asada", " ")
    print ("2. Sopa de pollo", " ") 
    print ("3. Lonja empanizada", " ")
    print (" ")
    comida2 = int(input("Respuesta: "))
    print (" ")
    if comida2 == 1:
        print ("Su orden es: Carne asada")
    elif comida2 ==2:
        print ("Su orden es: Sopa de pollo")
    elif comida2 ==3:
        print ("Su orden es: Lonja empanizada")
    else: print ("Valor ingresado incorrecto")

elif eleccion == 3:
    print ("Las opciones de cena son las siguientes: ", " ")
    print ("1. Platanos con frijoles molidos", " ")
    print ("2. Sushi", " ") 
    print ("3. Pupusas", " ")
    print (" ")
    comida3 = int(input("Respuesta: "))
    print (" ")
    if comida3 == 1:
        print ("Su orden es: Platanos con frijoles molidos")
    elif comida3 ==2:
        print ("Su orden es: sushi")
    elif comida3 ==3:
        print ("Su orden es: Pupusas")
    else: print ("Valor ingresado incorrecto")

elif eleccion == 4:
    print ("Las opciones de postres son las siguientes: ", " ")
    print ("1. Chessecake", " ")
    print ("2. Tres leches", " ") 
    print ("3. Postre de la casa", " ")
    print (" ")
    comida4 = int(input("Respuesta: "))
    print (" ")
    if comida4 == 1:
        print ("Su orden es: Chessecake")
    elif comida4 ==2:
        print ("Su orden es: Tres leches")
    elif comida4 ==3:
        print ("Su orden es: Portre de la casa")
    else: print ("Valor ingresado incorrecto")

elif eleccion == 5:
    print ("Las opciones de antojos son las siguientes: ", " ")
    print ("1. Alfajores", " ")
    print ("2. Dulcede leche", " ") 
    print ("3. Te Chai", " ")
    print (" ")
    comida5 = int(input("Respuesta: "))
    print (" ")
    if comida5 == 1:
        print ("Su orden es: Alfajores")
    elif comida5 ==2:
        print ("Su orden es: Dulce de leche")
    elif comida5 ==3:
        print ("Su orden")
    else: print ("Valor ingresado incorrecto")
elif eleccion == 6:
    print ("Programa finalizado")
else: print ("Valor ingresado incorreccto")

File: Menú.cpp
#include <iostream>
using namespace std;

int main (){
    int eleccion, comida, comida2, comida3, comida4, comida5, orden;
    
    cout << "¡Bienvenido!" << endl;
    proceso:
    cout << "Ingrese el numero de la izquierda acorde a la opcion que desea" << endl;
    cout << "1. Desayuno" << endl << "2. Almuerzo" << endl << "3. Cena" << endl << "4. Postres" << endl << "5. Antojos" << endl << "6. Salir" << endl << endl << "Respuesta: ";
    cin >> eleccion;
    cout << endl;

    if (eleccion == 1){
        cout << "Las opciones de desayuno son las siguientes: " << endl;
        cout << "1. Pupusas" << endl << "2. Huevos con frijoles y platanos" << endl << "3. Hot Cakes" << endl << endl << "Respuesta: ";
        cin >> comida;
        cout << endl;
        if (comida == 1){
            cout << "Su orden es: Pupusas";
        } else if (comida == 2){
            cout << "Su orden es: Huevos con frijojles y platanos";
        } else {cout << "Su orden es: Hot Cakes";}

    } else if (eleccion == 2){
        cout << "Las opciones de almuerzo son las siguientes: " << endl;
        cout << "1. Carne asada" << endl << "2. Sopa de pollo" << endl << "3. Lonja empanizada" << endl << endl << "Respuesta: ";
        cin >> comida2;
        cout << endl;
        if (comida2 == 1){
            cout << "Su orden es: Carne asada";
        } else if (comida2 == 2){
            cout << "Su orden es: Sopa de pollo";
        } else {cout << "Su orden es: Lonja empanizada";}

    } else if (eleccion == 3){
        cout << "Las opciones de cena son las siguientes: " << endl;
        cout << "1. Platanos con frijoles molidos" << endl << "2. Sushi" << endl << "3. Pupusas" << endl << endl << "Respuesta: ";
        cin >> comida3;
        cout << endl;
        if (comida3 == 1){
            cout << "Su orden es: Platano con frijoles molidos";
        } else if (comida3 == 2){
            cout << "Su orden es: Sushi";
        } else {cout << "Su orden es: Pupusas";}

    } else if (eleccion == 4){
        cout << "Las opciones de postres son las siguientes: " << endl;
        cout << "1. Chessecake" << endl << "2. Tres leches" << endl << "3. Postre de la casa" << endl << endl << "Respuesta: ";
        cin >> comida4;
        cout << endl;
        if (comida4 == 1){
            cout << "Su orden es: Chessecake";
        } else if (comida4 == 2){
            cout << "Su orden es: Tres leches";
        } else {cout << "Su orden es: Postre de la casa";}
        
    } else if (eleccion == 5){
        cout << "Las opciones de antojos son las siguientes: " << endl;
        cout << "1. Alfajores" << endl << "2. Dulce de leche" << endl << "3. Te Chai" << endl << endl << "Respuesta: ";
        cin >> comida5;
        cout << endl;
        if (comida5 == 1){
            cout << "Su orden es: Alfajores";
        } else if (comida5 == 2){
            cout << "Su orden es: Dulce de leche";
        } else {cout << "Su orden es: Te Chai";}
        
    } else if (eleccion == 6){
        cout << "Programa finalizado..." << endl;
        
    } else if (eleccion !=1 && eleccion !=2 && eleccion !=3 && eleccion !=4 && eleccion !=5 && eleccion !=6){
        cout << "El valor ingresado es incorrecto" << endl;
        cout << "¿Desea ordenar algo?" << endl;
        cout << "1. Si" << endl << "2. No" << endl << endl << "Respuesta: ";
        cin >> orden;
        cout << endl;
        if (orden == 1){
            goto proceso;
            cout << endl;
        } else {cout << "El programa finalizo...";}
    }
 
    return 0;
}